src/promptflow-evals/promptflow/evals/evaluators/_content_safety/flow/evaluate_with_rai_service.py

```diff
@@ -1,17 +1,24 @@
+import importlib.metadata
 import re
 import time
 from typing import List
+from urllib.parse import urlparse
 
 import numpy as np
 import requests
 from azure.core.credentials import TokenCredential
 from azure.identity import DefaultAzureCredential
 from constants import RAIService, Tasks
 from utils import get_harm_severity_level
-from urllib.parse import urlparse
 
 from promptflow.core import tool
 
+try:
+    version = importlib.metadata.version("promptflow-evals")
+except importlib.metadata.PackageNotFoundError:
+    version = "unknown"
```

Comment:
I think we should remove these few lines, because built-in evaluator is part of pf-evals so it shouldn't depend on that package again.

Reply:
removed